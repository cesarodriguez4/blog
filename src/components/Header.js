import React from 'react';
import { AppBar, Toolbar, Typography, IconButton } from 'material-ui';
import MenuIcon from 'material-ui-icons/Menu';

const bgHeader = {
  backgroundColor: 'white'
};

export default class Header extends React.Component {
	render() {
		return(
      <div>
        <AppBar style={bgHeader} position="static">
          <Toolbar>
            <IconButton aria-label="Menu">
              <MenuIcon />
            </IconButton>
            <Typography type="title">
              CesarJS | Blog
            </Typography>
          </Toolbar>
        </AppBar>
      </div>
		);
	};
}