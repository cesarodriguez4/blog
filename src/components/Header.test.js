import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';
import { MuiThemeProvider } from 'material-ui/styles';

it('render the AppBar correctly', () => {
  const head = renderer.create(
  	<MuiThemeProvider><Header></Header></MuiThemeProvider>).toJSON();
  expect(head).toMatchSnapshot();
});