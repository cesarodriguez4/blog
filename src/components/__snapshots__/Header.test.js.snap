// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the AppBar correctly 1`] = `
<div>
  <header
    className="MuiPaper-root-2758040674 MuiPaper-shadow4-941451405 MuiAppBar-root-2668177540 MuiAppBar-positionStatic-30597966 MuiAppBar-colorPrimary-15947162"
    style={
      Object {
        "backgroundColor": "white",
      }
    }
  >
    <div
      className="MuiToolbar-root-1760419744 MuiToolbar-gutters-344431791"
    >
      <button
        aria-label="Menu"
        className="MuiButtonBase-root-3177716317 MuiIconButton-root-3609702008"
        disabled={false}
        onBlur={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex="0"
        type="button"
      >
        <span
          className="MuiIconButton-label-2320069359"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-3510206624"
            focusable={false}
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-3868442396"
        />
      </button>
      <h2
        className="MuiTypography-root-233331062 MuiTypography-title-2106356098"
      >
        CesarJS | Blog
      </h2>
    </div>
  </header>
</div>
`;
